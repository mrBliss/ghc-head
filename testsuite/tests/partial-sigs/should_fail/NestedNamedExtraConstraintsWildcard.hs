{-# LANGUAGE RankNTypes, PartialTypeSignatures, NamedWildcards #-}
module NestedNamedExtraConstraintsWildcard where

foo :: Bool -> (Eq a, _a) => a
foo = undefined
