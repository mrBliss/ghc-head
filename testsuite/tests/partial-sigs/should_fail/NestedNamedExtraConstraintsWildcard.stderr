
NestedNamedExtraConstraintsWildcard.hs:4:23:
    Invalid partial type signature:
    Bool -> (Eq a, _a) => a
    A named wildcard cannot occur as a constraint
